### DISCUSSION TITLE
Grouped collections / globals

### DISCUSSION THREAD

**Discussion by edblocker** [payloadcms/payload#507] [Q&A] [answered]
Hi,

I was wondering if it's possible to have some sort of grouped collection or globals? I'm stumbling across this problem that the amount of collections is really confusing. To make it more clear I'm prefixing collections with the type of collection they're belonging to, however this behaviour is really unwanted. So I was wondering if I missed something or whether this "feature" is in the pipeline.

Kind regards,
Eddy

**Comment by DanRibbens** [comment, marked as answer]
Hey @edblocker,
Great question, you didn't miss anything. As of today, the built in nav doesn't offer any structure. You could make a custom Nav component, though, I'm in favor of building this one in as a feature to Payload.

Give us some time here to circle back around on it...


If you want to build your own, check out the docs for customizing the admin UI: https://payloadcms.com/docs/admin/components
`Nav` is the one you will need to replace.

A good starting point would be to copy the existing Nav component to build on top of: https://github.com/payloadcms/payload/tree/master/src/admin/components/elements/Nav

It isn't ideal, but you could use your prefixes to group them or something, up to you on how you'd handle it.
Let us know if you do build something and want to share.

Thanks,
Dan

**Reply by Rar9** [reply]
@DanRibbens This feels wrong that everyone should create a custom nav to resolve the issue. 

Please consider adding a 2nd level, so category can have deeper categories.

**Reply by DanRibbens** [reply]
> @DanRibbens This feels wrong that everyone should create a custom nav to resolve the issue. 
> 
> Please consider adding a 2nd level, so category can have deeper categories. 

You're right @Rar9! This answer is out of date. 

There is now the `admin.group` property that allows for organizing nav items under names that you provide. You no longer need a custom nav component for this.

It is documented here: https://payloadcms.com/docs/configuration/collections#admin-options
